### ISSUE TITLE
Support Number Component

### ISSUE THREAD

**Issue by Azador** [DrozmotiX/ioBroker.esphome#132]
Support of the number component of ESPHome would be great.

**Comment by Azador** [comment]
Or, at least, allow to use the number component without breaking the api connection to the esphome adapter.
Right now, when I try to use the number component, the adapter cannot connect to the device and therefore all other components cannot be used.

**Comment by 4eversr** [comment]
Yeah, i ran into the same problem when using the "number"-type in the yaml of the esp.

See here: https://forum.iobroker.net/topic/44522/benutzen-von-esphome-an-einem-beispiel-update-05-01-22/107
